submitit/slurm/slurm.py

```diff
@@ -190,7 +190,7 @@ class SlurmJobEnvironment(job_environment.JobEnvironment):
 
     def _requeue(self, countdown: int) -> None:
         jid = self.job_id
-        subprocess.check_call(["scontrol", "requeue", jid])
```

Comment:
on some cluster `scontrol requeue` seems to become a zombie process, this is an additional safeguard to make sure it times out if it's not able to complete. Still, this should have been blocking so unsure how this could have happened.

Reply:
good for me.